src/loginServices.js

```diff
@@ -104,6 +104,9 @@ function canReadConfig(store) {
  */
 export function loadTranslations(store, locale, defaultTranslations = {}) {
   const parentLocale = locale.split('-')[0];
+  // Since moment.js don't support translations like it or it-IT-u-nu-latn
+  // we need to build string like it_IT for fetch call
+  const loadedLocale = locale.replace('-', '_').split('-')[0];
```

Comment:
I don't think this line does what you want it to, because first it'll replace dashes with underscores, then split on dashes (there will now be none), then return the first element, which means it'll be identical to the `locale` argument. Thus function needs to care about four different values:
1. locale including region and number scheme if present, separated by dashes, e.g. `fr` or `fr-FR` or `fr-FR-u-nu-latn`, because we want to call `dispatch(setLocale(locale))` with a full string in order for react-intl to get the numbering system so it can handle number formatting correctly and for the document's `lang` attribute. 
2. locale including region if present, separated by dashes, e.g. `fr` or `fr-FR` because that's the kind of value `getLangDir` and `moment` expect. 
3. locale without the region, e.g. `fr`, because if `moment` doesn't recognize sth like `en-SE` we want it to fall back to loading `en`. 
4. locale including region if present, separated by underscore, e.g. `fr` or `fr_FR` because that's the kind of value our sad sad lokalise system uses as a key into the `translations` array imported from `stripes-config`.

Reply:
So @zburke what I need to change ? I don't have any ideas for now